app/model/application.js

```diff
@@ -79,12 +87,16 @@ ApplicationSchema.statics.fetchReleases = function(application) {
 }
 
 ApplicationSchema.statics.parseAppHubFileIfPossible = function(application) {
-  return Promise.try(() => {
-    // Parse the .desktop file
-    const appHubFileBuf = new Buffer(application.appHubFileResult.content, 'base64');
-    const appHubData = JSON.parse(appHubFileBuf.toString());
-    application.priceUSD = appHubData.priceUSD;
-    delete application.appHubFileResult;
+  const fullName = application.github.fullName;
+  return gh.request(`GET /repos/${fullName}/contents/.apphub`, {
```

Comment:
I think it actually makes sense to show explicit errors not only for this function but also the desktop file.